python/requirements.txt

```diff
@@ -20,4 +20,7 @@ pyspark==3.0.1
 tensorflow-probability==0.15.0
 
 # DAG visualization
-pygraphviz==1.7
\ No newline at end of file
+pygraphviz==1.7
```

Comment:
@mohazahran Why aren't you changing `scipy` version here?

Reply:
It's a Pingoun dependency

Reply:
Fix it to a minor version too.